### PR TITLE
Vijayasree/ refactor: update who-we-are to new-show-component

### PR DESCRIPTION
**Pull request by vijayasree-deriv**
Changes:

-   ...

## Type of change

-   [ ] Bug fix
-   [ ] New feature
-   [ ] Update feature
-   [ ] Refactor code
-   [ ] Translation to code
-   [ ] Translation to crowdin
-   [ ] Script configuration
-   [ ] Improve performance
-   [ ] Style only
-   [ ] Dependency update
-   [ ] Documentation update
-   [ ] Release

### CODE REVIEW

File: src/pages/who-we-are/_OurOffices.tsx [hunk 1]
```diff
@@ -173,7 +173,7 @@ const OurOffices = () => {
             </StyledHeader>
 
             <Flex>
-                <Show.Desktop max_width="tabletL">
+                <Desktop max_width="tabletL">
```
Comment (sean-binary) [inline src/pages/who-we-are/_OurOffices.tsx:176]: I don't think `Desktop` and `Mobile` uses `max_width` props. Please take a look at the `visibility.tsx` file.

**Review by sean-binary — CHANGES_REQUESTED** [review]

File: src/pages/who-we-are/_OurOffices.tsx (hunk 1 above)
Comment (vijayasree-deriv) [reply src/pages/who-we-are/_OurOffices.tsx:176]: The Desktop component renders a DesktopLayer component which has a media query where max_width uses a breakpoint value. (Similarly with Mobile component , MobileLayer and min_width.)

Comment (sean-binary) [reply src/pages/who-we-are/_OurOffices.tsx:176]: Yep but the prop should be `breakpoint` in that case. This is still working as the default of the `Desktop` component is `tabletL`.

Comment (vijayasree-deriv) [reply src/pages/who-we-are/_OurOffices.tsx:176]: yes you are right, I will make the necessary changes and update.

**Review by sean-binary — APPROVED** [review]

**Review by sean-binary — APPROVED** [review]

**Pull request merged**